### PR TITLE
Fix Broken Links and Typos 

### PR DESCRIPTION
**Pull request by SourabhSaraswat-191939**

### CODE REVIEW

File: README.md
Comment (vchrombie) [inline README.md:13]: Can you please change the link text of the dashboard?
```
- [OPNFV Development Dashboard](https://chaoss.biterg.io)
+ [CHAOSS Development Dashboard](https://chaoss.biterg.io)
```

File: basics/scenarios.md
Comment (vchrombie) [inline basics/scenarios.md:14]: I think you might need to revert these changes. The docs are actually generated using Jekyll and it is accessible at [chaoss.github.io/grimoirelab-tutorial](https://chaoss.github.io/grimoirelab-tutorial/). The links work fine just the way they are now.

File: basics/scenarios.md
Comment (vchrombie) [inline basics/scenarios.md:79]: same as above

File: basics/scenarios.md
Comment (vchrombie) [inline basics/scenarios.md:87]: same as above

File: basics/scenarios.md
Comment (vchrombie) [inline basics/scenarios.md:96]: same as above

File: gelk/intro.md
Comment (vchrombie) [inline gelk/intro.md:23]: same as above

File: basics/components.md
Comment (vchrombie) [inline basics/components.md:19]: Thanks for pointing the broken links but they have fixed differently. https://chaoss.github.io/grimoirelab-tutorial/basics/components.html

All the links should end with `.html` and `grimoirelab-tutorial/` shouldn't be deleted from the link.
It would be great if you can update the required changes.

File: basics/components.md
Comment (vchrombie) [inline basics/components.md:23]: same as above

**Review by vchrombie — CHANGES_REQUESTED** [review]
Hi @SourabhSaraswat-191939, thanks for the PR but I think we need to fix a few issues with the changes. Please check my inline comments and let us know if you need any help.

You can keep adding commits to the same branch. Once, the PR is perfect, we can squash them.
Thanks.

File: README.md
Comment (SourabhSaraswat-191939) [reply README.md]: Okay, sure .

File: basics/components.md
Comment (SourabhSaraswat-191939) [reply basics/components.md]: okay  Sir

File: basics/scenarios.md
Comment (SourabhSaraswat-191939) [reply basics/scenarios.md]: Okay Sir , 
but "chaoss.github.io/grimoirelab-tutorial " this is hosted on Github Pages and on this repository everything is in .md format and Github pages serves their repository live content , then how and when it is converting to .html from .md extension using Jekyll .
It can be a silly question , but i got confused in this .

Comment (vchrombie) [reply basics/scenarios.md]: Jekyll converts these pages from md to html files and they are served.

For example, https://github.com/chaoss/grimoirelab-tutorial/blob/master/python/scripting.md is converted to https://chaoss.github.io/grimoirelab-tutorial/python/scripting.html

Also, you need not address me as sir :slightly_smiling_face:

File: basics/components.md
Comment (vchrombie) [inline basics/components.md:17]: I found one more broken link here, it would be great if you can update this too.

```
- /scenarios.md#
+ /scenarios.html#
```

**Review by vchrombie — CHANGES_REQUESTED** [review]
Hi @SourabhSaraswat-191939, I think we missed this. Can you please work on this too?

File: basics/scenarios.md
Comment (SourabhSaraswat-191939) [reply basics/scenarios.md]: Now I understand and also readed some documents related to Jekyll  .

Comment (SourabhSaraswat-191939) [reply basics/scenarios.md]: And i did everything as per your Instructions , to make this PR ready to merge . If anything left please tell me i will do that .

Comment (vchrombie) [reply basics/scenarios.md]: I think you have missed this https://github.com/chaoss/grimoirelab-tutorial/pull/153#discussion_r598103536
Let me know if it is not clear.

Comment (SourabhSaraswat-191939) [reply basics/scenarios.md]: Ohh , sorry i missed that one . 
Give me a second  to let me fix that

File: basics/components.md
Comment (SourabhSaraswat-191939) [reply basics/components.md]: Done

**Comment by vchrombie** [comment]
Thanks for your work. The changes look great.

As a final step, it would be great if you squash the commits into one and have a good commit message exaplaing about the changes.
Please refer to this post [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).

Let me know if you need help.

**Comment by SourabhSaraswat-191939** [comment]
i think i did something wrong , while squashing .
I used this steps :
git rebase -i HEAD~9
then changed pick to s in commits except in 1 commit
then i wrote commit message 
and after that i used git merge .


@vchrombie  Please help me in this .

File: tools-and-tips/html5-app-latest-activity.md
Comment (vchrombie) [inline tools-and-tips/html5-app-latest-activity.md:13]: Can you change it to a relative link, just like the other links?

[Producing Kibana dashboards](../gelk/intro.html).

**Review by vchrombie — CHANGES_REQUESTED** [review]
The commits now look good, one final comment on the changes.
You can add the new change to the same commit using `--amend`.
Thanks.

File: tools-and-tips/html5-app-latest-activity.md
Comment (SourabhSaraswat-191939) [reply tools-and-tips/html5-app-latest-activity.md:13]: Done 👍🏻

**Comment by SourabhSaraswat-191939** [comment]
@vchrombie is there any changes needed in this PR to get merged ?

**Comment by SourabhSaraswat-191939** [comment]
> @vchrombie is there any changes needed in this PR to get merged ?

@vchrombie Waiting for your reply .

**Comment by vchrombie** [comment]
Hi @SourabhSaraswat-191939, sorry for the late reply. I think I missed your comment.

The changes look good to me. But I think you need to improve the commit message and it would be good if you can add a commit description that briefly explains the change.

Thanks.

**Comment by SourabhSaraswat-191939** [comment]
> The changes look good to me. But I think you need to improve the commit message and it would be good if you can add a commit description that briefly explains the change.

Sure , i will do that soon .

**Comment by SourabhSaraswat-191939** [comment]
@vchrombie i updated commit title and description . If still anything left please let me know .

**Comment by vchrombie** [comment]
Hi @SourabhSaraswat-191939

It is advisable to break the commit description into multiple lines if it is too long. Also, you can change the commit description to something which explains the changes very briefly. You need not explain each change. Please check my suggestion.

```
Fix broken links, typos

This commit updates the broken links and also fixes a few typos.
<sign-off>
```

Thanks.

**Comment by SourabhSaraswat-191939** [comment]
> It is advisable to break the commit description into multiple lines if it is too long. Also, you can change the commit description to something which explains the changes very briefly. You need not explain each change. Please check my suggestion.

Done 👍🏻  . If still anything left please let me know .

**Comment by vchrombie** [comment]
> If still anything left please let me know .

The body of the PR is too long, can you break it into two lines or reduce the length. Ideally, the body should be wrapped at 72 characters. In case if it is too long, please break the commit message into multiple lines.

It would be great if you can follow the [commit guidelines](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md#guidelines-to-follow-to-write-good-commit-messages) given in the contributing doc for further contributions.

**Comment by SourabhSaraswat-191939** [comment]
> The body of the PR is too long, can you break it into two lines or reduce the length. Ideally, the body should be wrapped at 72 characters. In case if it is too long, please break the commit message into multiple lines.

Done this changes .
If still anything left please let me know .

**Review by vchrombie — APPROVED** [review]
LGTM

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**
